e2e/cypress/integration/collapsed_reply_threads/crt_settings_spec.js

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+describe('CRT Settings', () => {
+    let testTeam;
+    let testChannel;
+
+    before(() => {
+        cy.apiInitSetup({loginAfter: true, promoteNewUserAsAdmin: true}).then(({team, channel}) => {
```

Comment:
Could the test user be a regular user, too? If yes, then please change and only use admin user as required.

Reply:
I was meaning to ask about that, 
when I run the collapsed_reply_threads test suite, consecutive tests would fail with permission errors, 
e.g getting preferences for a user resulted in a 401 or 403 response. 
Making the user as admin solved that but I never understood why.
Running each test individually worked though.

Reply:
It probably happened when a regular user is creating another user. However, in this case, `promoteNewUserAsAdmin: true` can be omitted.

Reply:
@saturninoabril I've changed only this test case but I keep getting an error on subsequent tests...

<img width="445" alt="Screenshot 2021-08-16 at 2 31 58 PM" src="https://user-images.githubusercontent.com/3829551/129557113-1733e501-ba1d-4d17-9e7a-c5d0a43e8564.png">

Reply:
If I run each test individually they will pass.

Reply:
Thanks for updating!
Oh, I see. It's because running such way will execute the [global before hook](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/support/index.js#L108) once only. We normally run test per [spec file](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/run_test_cycle.js#L63).